Documentation/Contributors/BuildGuide/README.md

```diff
@@ -1,10 +1,34 @@
 # Build Guide
 
+- [Quickstart](#quickstart)
 - [Get the Code](#get-the-code)
 - [Build the Code](#build-the-code)
 - [Build Scripts](#build-scripts)
 - [Travis and Continuous Integration](#travis-and-continuous-integration)
 
+## Quickstart
+
+1. [Clone the Reop](#get-the-code)
```

Comment:
```suggestion
1. [Clone the Repository](#get-the-code)
```

"Repo" is short for "repository."  I just realized that the abbreviated version may not be clear to everyone.

Reply:
I say repo so often I hadn't thought of that! Good catch.